lib/modules/mod_filestore_svn.phtml

```diff
@@ -202,11 +209,16 @@
 				$this->get_path($svn, $name)
 			);
 			$svn['instance']->cat->_prepared = false;
-			$result = $svn['instance']->cat->run($args, $svn['switches']);
+			try {
+				$result = $svn['instance']->cat->run($args, $svn['switches']);
+			} catch(VersionControl_SVN_Exception $e) {
+				return ar::error('svn exception','svn',$e);
```

Comment:
Better to use a numeric code instead of 'svn'. Just make sure it's not used already. (Exceptions must have an integer code, ar:error wil cast this, but this means the code will become '0')